go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -2016,6 +2018,9 @@ func (s *VtctldServer) GetTablets(ctx context.Context, req *vtctldatapb.GetTable
 
 		tablets := make([]*topodatapb.Tablet, 0, len(tabletMap))
 		for _, ti := range tabletMap {
+			if req.TabletType != 0 && ti.Type != req.TabletType {
```

Comment:
why is this `0` and not `UNKNOWN`?

Reply:
Only because I copied the existing check/filter that was already used elsewhere in the function.🙂 I wish that I had changed both locations instead though. Whomever changes this file next should do that.

Reply:
Since it's a non-functional change I snuck it in here:
https://github.com/vitessio/vitess/pull/14315/commits/459a27b7d6b5313521bac8dad4f2564ca527ef34